src/app/courses/request/coursesrequest.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'planet-courses',
+  templateUrl: 'coursesrequest.component.html',
+  styleUrls: ['coursesrequest.component.scss']
```

Comment:
Since you are using the same styles as the add component, you can import the add component stylesheet instead of recreating them same file.

Reply:
Yes I agree, a dash should be used for the courses-add files. For now I was just following the existing format used in the `meetupsadd` component. As for the courses-request styles, it is just a placeholder at the moment. There is nothing in the html file either.

Reply:
I see. With so many components for Courses, should it be made into a feature module, @paulbert?

Reply:
Yes, I agree both to the dash notation and converting this to a feature module.

As for folder naming, I think we can go with a `<descriptor>-<feature>` format rather than the opposite which the Angular style guide seems to suggest.  That way we can have folder names which will auto complete with TAB when navigating in the shell.

Reply:
Yep you are right @paulbert. So the folder names should be opposite to what I said like the `add` folder should be renamed to `courses-add`